packages/components/descriptions/src/descriptions.ts

```diff
@@ -0,0 +1,44 @@
+import { buildProps, definePropType } from '@element-plus/utils/props'
+import { isValidComponentSize } from '@element-plus/utils/validators'
+import type { ComponentSize } from '@element-plus/utils/types'
+import type { ExtractPropTypes } from 'vue'
+
+export const descriptionsProps = buildProps({
+  border: {
+    type: Boolean,
+    default: false,
+  },
+  column: {
+    type: Number,
+    default: 3,
+  },
+  direction: {
+    type: definePropType<'horizontal' | 'vertical'>(String),
+    default: 'horizontal',
+  },
+  size: {
```

Comment:
```ts
import { componentSizes } from '@element-plus/constants'
{
  values: componentSizes
}
```

Reply:
Note that after using `values` you don't have to put validator here since `values` will generate a validator for us